src/connector/postgres.rs

```diff
@@ -254,6 +264,8 @@ impl PostgresUrl {
         let mut pool_timeout = Some(Duration::from_secs(10));
         let mut pg_bouncer = false;
         let mut statement_cache_size = 500;
+        let mut max_connection_lifetime = None;
+        let mut max_idle_connection_lifetime = None;
```

Comment:
```suggestion
        let mut max_idle_connection_lifetime = Some(Duration::from_secs(300));
```

See: https://github.com/prisma/prisma-engines/blob/master/query-engine/connectors/sql-query-connector/src/database/postgresql.rs